routes/index.js

```diff
@@ -196,39 +196,73 @@ module.exports = function (passport) {
   });
 
 
-  router.get('/userManagement/:badge', checkAuth, function(req,res){
+  router.post('/userManagement/promoteUser/:badge',checkAuth, function (req, res) {
+    if (!req.body) {
+      return res.sendStatus(400);
+    }
+    var BagdeID = req.params.badge;
+    dbAPI.modifyUser(BagdeID, { status: "Manager" }).then(function (result) {
+      if (result == undefined) {
+        console.log("Error");
+      }
+      else {
+        console.log("promotion changed successfully");
+      }
+    });
+    res.redirect('/userManagement');
+  });
+
+
+  router.post('/userManagement/demoteManager/:badge',checkAuth, function (req, res) {
+    if (!req.body) {
+      return res.sendStatus(400);
+    }
+    var BagdeID = req.params.badge;
+    dbAPI.modifyUser(BagdeID, { status: "User" }).then(function (result) {
```

Comment:
Before modifying this should check first that:
1) The user we are modifying actually has the status 'Manager'
2) Is also not an 'Admin'

Reply:
I do not know is there a way to get a specific data of the user as status ?

Reply:
Use `validateUser()`?

Reply:
I guess we do not need these extra as it is already checked in the UI, if you go to an admin you won't see any of these options.

Reply:
That is true. But that API route could potentially be reused somewhere else and we need to ensure that no assumptions can be made. Doing that check again makes it more secure.

Reply:
I am almost done with it, I am just doing some more tests to make sure everything is going well.

Reply:
Sounds good. Thanks.